onnxruntime/core/framework/allocation_planner.cc

```diff
@@ -100,19 +100,30 @@ std::ostream& operator<<(std::ostream& out, std::pair<const SequentialExecutionP
   return out;
 }
 
+static const KernelCreateInfo& GetKernel(
+    const std::unordered_map<NodeIndex, gsl::not_null<const KernelCreateInfo*>>& kernel_create_info_map,
+    NodeIndex node_index) {
+  auto entry = kernel_create_info_map.find(node_index);
+  ORT_ENFORCE(entry != kernel_create_info_map.cend(),
```

Comment:
nit: any way we can provide the node op type/name to help with debugging if we hit this error ?

Reply:
Not an easy way to get it, but this should be pretty impossible to fail. SessionState enforces there being an entry for every valid node, and we are iterating the valid nodes here to get the kernel create info.